overrides/config/forestry/backpacks.cfg

```diff
@@ -100,10 +100,21 @@ backpacks {
                 minecraft:yellow_flower:0
                 evilcraft:undead_log
                 integrateddynamics:menril_log
+                plants2:ambrosia_a
+                plants2:apocynum_c
+                plants2:akebia_q
+                plants2:amarthus_h_seeds
```

Comment:
typo, should be plants:amaranthus_h_seeds

Reply:
Addressed by d92ece4.